exchange/factory.go

```diff
@@ -55,7 +61,7 @@ func (fty *Factory) NewExchange(resp *Response, vp ValidPeriod, validityURL *url
 		Date:        vp.Date(),
 		Expires:     vp.Expires(),
 		Certs:       certutil.GetCertificates(fty.CertChain),
-		CertUrl:     fty.CertURL,
+		CertUrl:     u.ResolveReference(fty.CertURL),
```

Comment:
CertUrl must be HTTPS. As it's the spec requirement, should we ensure it rather than making it caller's responsibility?

Reply:
Sorry I overlooked that point. Thanks for your exploration!